com.unity.robotics.urdf-importer/Runtime/AssetHandlers/UrdfAssetPathHandler.cs

```diff
@@ -87,27 +89,21 @@ public static string GetFullAssetPath(string relativePath)
 
         public static string GetRelativeAssetPathFromUrdfPath(string urdfPath, bool convertToPrefab=true)
         {
-            if (!urdfPath.StartsWith(@"package://"))
-            {
-               Debug.LogWarning(@$"{urdfPath} is not a valid URDF package file path. Path should start with package://, and URDF file should be in the directory root.");
-               if (urdfPath.Substring(0, 3) == "../")
```

Comment:
We need to keep the logic here. I was testing importing niryo one and it threw error without the relative path handling here.

Reply:
OK, I return these lines at https://github.com/Unity-Technologies/URDF-Importer/pull/147/commits/09603fca7ce91fc4ac555e1ff1e85ba6390aa8b7